pkg/controller/postgresqldatabase/postgresqldatabase_controller.go

```diff
@@ -0,0 +1,235 @@
+package postgresqldatabase
+
+import (
+	"context"
+	"database/sql"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	"github.com/lib/pq"
+	lunarwayv1alpha1 "go.lunarway.com/postgresql-controller/pkg/apis/lunarway/v1alpha1"
+
```

Comment:
Newline can be removed so all external imports are grouped and sorted accordingly.